src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -263,11 +264,7 @@ const CreateBenchmarkPage = () => {
                       required
                       type="number"
                       name="baseline_level"
-                      value={
-                        benchmarkFormState["baseline_level"] !== ""
-                          ? benchmarkFormState["baseline_level"]
-                          : ""
-                      }
```

Comment:
While this expression for `value` doesn't generate any warnings, I don't think it's actually doing what's intended. It's effectively the same as `value={benchmarkFormState["baseline_level"]}` without any ternary operator logic. I replaced this (and other similar instances) with a simpler expression that will use an empty string if `benchmarkFormState["baseline_level"]` is falsy (null or undefined).

I'm happy to revert this change if there's a reason for the original code that I missed.